### ISSUE TITLE
PROJECT: github-pr-labeler

### ISSUE THREAD

**Issue by MichaelClifford** [operate-first/support#383]
### Select all that apply

- [ ] Create a new ArgoCD project for existing team
- [X] Enlist an application to be deployed automatically via ArgoCD
- [ ] Add a new user to an ArgoCD Project

### Team name or your Argo CD project

aicoe

### Application source git repository

https://github.com/aicoe-aiops/github-labeler

### Path to manifests within the repository

/manifests/base

### Target cluster

Zero

### Target namespace for the application

github-labeler

**Comment by MichaelClifford** [comment]
cc @antter

**Comment by MichaelClifford** [comment]
related to https://github.com/operate-first/support/issues/369

Will requests targeting the zero cluster be moved to smaug?

**Comment by 4n4nd** [comment]
> Will requests targeting the zero cluster be moved to smaug?

yes, most zero requests should be moved to the smaug cluster. Unless there are other reasons for them to use emea/rick.
